packages/xo-web/src/xo-app/xostor/new-xostor-form.js

```diff
@@ -235,18 +240,59 @@ const NetworkCard = decorate([
     computed: {
       networksPredicate: (state, props) => network => {
         const isOnPool = network.$pool === state.poolId
-        const pifs = network.PIFs
+        const pifs = props.pifsByNetwork[network.id]
         return state.onlyShowXostorNetworks
-          ? isOnPool && pifs.length > 0 && pifs.every(pifId => props.pifs[pifId].ip !== '')
+          ? isOnPool &&
+              doesNetworkHasPifs(network) &&
+              arePifsStatic(pifs) &&
+              doesPifsHaveIp(pifs) &&
+              arePifsAttached(pifs)
           : isOnPool
       },
+      networkHasPifs: (state, props) => doesNetworkHasPifs(props.networks[state.networkId]),
+      pifsAreAttached: (state, props) => arePifsAttached(props.pifsByNetwork[state.networkId]),
+      pifsAreStatic: (state, props) => arePifsStatic(props.pifsByNetwork[state.networkId]),
+      pifsHaveIp: (state, props) => doesPifsHaveIp(props.pifsByNetwork[state.networkId]),
+      isInterfaceNameContainsWhiteSpace: state => state.interfaceName.includes(' '),
+      isInterfaceNameReserved: state => state.interfaceName.trim() === 'default',
+      isNetworkCompatible: state =>
```

Comment:
I think there should be a text or a tooltip giving this info before user try to save it